documentation/modules/new-migrating-virtual-machines-cli.adoc

```diff
@@ -0,0 +1,1080 @@
+----
+$ cat << EOF | {oc} apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: <secret>
+  namespace: <namespace>
+  ownerReferences: <1>
+    - apiVersion: forklift.konveyor.io/v1beta1
+      kind: Provider
+      name: <provider_name>
+      uid: <provider_uid>
+  labels:
+    createdForProviderType: vsphere
+    createdForResourceType: providers
+type: Opaque
+stringData:
+  user: <user> <2>
+  password: <password> <3>
+  insecureSkipVerify: <true/false> <4>
+  cacert: | <5>
+    <ca_certificate>
+  url: <api_end_point> <6>
+EOF
+----
+<1> The `ownerReferences` section is optional.
+<2> Specify the vCenter user.
+<3> Specify the user password.
+<4> Specify `<true>` to skip certificate verification, specify `false` to verify the certificate. Defaults to `false` if not specified. Skipping certificate verification proceeds with an insecure migration and then the certificate is not required. Insecure migration means that the transferred data is sent over an insecure connection and potentially sensitive data could be exposed.
+<5> Enter the CA certificate for the server that serves the SDK, either vCenter or ESXi, depending on the configuration of the provider, unless the CA certificate was replaced by a third-party certificate, in which case, enter the {manager} Apache CA certificate. You can retrieve the {manager} CA certificate at https://<engine_host>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA.
+<6> Specify the API end point URL, for example, `https://<vCenter_host>/sdk`.
+endif::[]
+ifdef::rhv[]
+[source,yaml,subs="attributes+"]
+----
+$ cat << EOF | {oc} apply -f -
+apiVersion: v1
+kind: Secret
+metadata:
+  name: <secret>
+  namespace: <namespace>
+  ownerReferences: <1>
+    - apiVersion: forklift.konveyor.io/v1beta1
+      kind: Provider
+      name: <provider_name>
+      uid: <provider_uid>
+  labels:
+    createdForProviderType: ovirt
+    createdForResourceType: providers
+type: Opaque
+stringData:
+  user: <user> <2>
+  password: <password> <3>
+  insecureSkipVerify: <true/false> <4>
+  cacert: | <5>
+    <ca_certificate>
+  url: <api_end_point> <6>
+EOF
+----
+<1> The `ownerReferences` section is optional.
+<2> Specify the {rhv-short} {manager} user.
+<3> Specify the user password.
+<4> Specify `<true>` to skip certificate verification, specify `false` to verify the certificate. Defaults to `false` if not specified. Skipping certificate verification proceeds with an insecure migration and then the certificate is not required. Insecure migration means that the transferred data is sent over an insecure connection and potentially sensitive data could be exposed.
+<5> Enter the CA certificate for the API server of the OpenShift cluster, unless it was replaced by a third-party certificate, in which case, enter the {manager} Apache CA certificate. You can retrieve the {manager} CA certificate at https://<engine_host>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA.
```

Comment:
I think you changed the description of the server that is mentioned here by mistake - it shouldn't be `of the OpenShift cluster` in case of RHV

Reply:
Fixed.